scripts/dashboard/Activity.vue

```diff
@@ -24,6 +24,7 @@ let txCount = 0;
 const updating = ref(false);
 const isHistorySynced = ref(false);
 const rewardAmount = ref(0);
+let nRewardUpdateHeight = 0;
```

Comment:
I think we should also add
 `nRewardUpdateHeight = 0;`

in the reset() function

Reply:
Good catch, added.